catalog/spatial/registry/registry-identification-plugin/src/main/java/org/codice/ddf/registry/identification/IdentificationPlugin.java

```diff
@@ -91,10 +124,70 @@ public CreateRequest process(CreateRequest input)
     @Override
     public UpdateRequest process(UpdateRequest input)
             throws PluginExecutionException, StopProcessingException {
+
+        OperationTransaction operationTransaction = (OperationTransaction) input.getProperties()
+                .get(Constants.OPERATION_TRANSACTION_KEY);
+
+        if (operationTransaction == null) {
+            throw new UnsupportedOperationException(
+                    "Unable to get OperationTransaction from UpdateRequest");
+        }
+
+        if (Requests.isLocal(input)) {
+            List<Metacard> previousMetacards = operationTransaction.getPreviousStateMetacards();
+
+            boolean transientAttributeUpdates = false;
+            if (input.getProperties()
+                    .get(RegistryConstants.TRANSIENT_ATTRIBUTE_UPDATE) != null) {
+                transientAttributeUpdates = true;
+            }
+
+            Map<String, Metacard> previousMetacardsMap = new HashMap<>();
+            for (Metacard metacard : previousMetacards) {
+                previousMetacardsMap.put(metacard.getId(), metacard);
+            }
+
+            ArrayList<Map.Entry<Serializable, Metacard>> entriesToRemove = new ArrayList<>();
+
+            for (Map.Entry<Serializable, Metacard> entry : input.getUpdates()) {
+                Metacard updateMetacard = entry.getValue();
+                Metacard existingMetacard = previousMetacardsMap.get(updateMetacard.getId());
+
+                if (existingMetacard != null) {
+                    if (existingMetacard.getAttribute(RegistryObjectMetacardType.REGISTRY_ID)
+                            .getValue()
+                            .equals(updateMetacard.getAttribute(RegistryObjectMetacardType.REGISTRY_ID)
+                                    .getValue())) {
+
+                        if (transientAttributeUpdates || updateMetacard.getModifiedDate()
```

Comment:
Remove the transient part of this if and change the other check to be something like `!updateMetacard.getModifiedDate().before(existingMetacard.getModifiedDate()`